packages/dataviews-client/src/resources/resources-endpoint.ts

```diff
@@ -0,0 +1,55 @@
+import { DatasetTypes, Resource } from '@globalfishingwatch/api-types'
```

Comment:
The same comment about naming, if we change the method name I'd change the file name too